<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-17T15:09:40 IST" name="petsuite" finished-at="2025-09-17T15:10:02 IST" duration-ms="21798">
    <groups>
    </groups>
    <test started-at="2025-09-17T15:09:40 IST" name="Test" finished-at="2025-09-17T15:10:02 IST" duration-ms="21798">
      <class name="api.test.userTest2">
        <test-method is-config="true" signature="setup()[pri:0, instance:api.test.userTest2@1190200a]" started-at="2025-09-17T15:09:40 IST" name="setup" finished-at="2025-09-17T15:09:41 IST" duration-ms="201" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testPostUser()[pri:1, instance:api.test.userTest2@1190200a]" started-at="2025-09-17T15:09:41 IST" name="testPostUser" finished-at="2025-09-17T15:09:47 IST" duration-ms="6309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostUser -->
        <test-method signature="testgetuserbyname()[pri:2, instance:api.test.userTest2@1190200a]" started-at="2025-09-17T15:09:47 IST" name="testgetuserbyname" finished-at="2025-09-17T15:09:51 IST" duration-ms="3925" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.test.userTest2.testgetuserbyname(userTest2.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testgetuserbyname -->
        <test-method signature="testUpdateuserbyname()[pri:3, instance:api.test.userTest2@1190200a]" started-at="2025-09-17T15:09:51 IST" name="testUpdateuserbyname" finished-at="2025-09-17T15:10:00 IST" duration-ms="9366" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateuserbyname -->
        <test-method signature="testdeleteuserbyusername()[pri:4, instance:api.test.userTest2@1190200a]" started-at="2025-09-17T15:10:00 IST" name="testdeleteuserbyusername" finished-at="2025-09-17T15:10:02 IST" duration-ms="1638" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testdeleteuserbyusername -->
      </class> <!-- api.test.userTest2 -->
    </test> <!-- Test -->
  </suite> <!-- petsuite -->
</testng-results>
